include/revng/Pypeline/Helpers/Python/RunnerInfo.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "nanobind/nanobind.h"
+
+namespace revng::pypeline::helpers::python {
+
+// Helper class to unpack containers from a nanobind::list
+// To be used in conjunction with PipeRunner or AnalysisRunner
```

Comment:
If you have a multiline comment where the second line has a capital letter, the previous line must end with a `.`.